exercises/concept/regular-chatbot/regular-chatbot.spec.js

```diff
@@ -0,0 +1,85 @@
+import {
+  isValidCommand,
+  removeEmoji,
+  checkPhoneNumber,
+  getURL,
+  niceToMeetYou,
+} from './regular-chatbot';
+
+describe('isValidCommand', () => {
+  test('recognizes wheter the command is at the first position', () => {
+    expect(isValidCommand("Chatbot, play a song from the 80's.")).toBe(true);
+    expect(isValidCommand('Hey Chatbot, where is the closest pharmacy?')).toBe(
+      false
+    );
+  });
```

Comment:
Comment for the whole file: Make sure all test cases are different from the examples in the instruction file.

Reply:
I didn't think about that, thanks for pointing it out 😅